lib/src/dom/within.dart

```diff
@@ -34,7 +32,7 @@ class WithinQueries extends ScopedQueries {
   WithinQueries._(this.container) : super(() => container);
 
   /// The node within which the queries are scoped.
-  final Node container;
+  final Node? container;
```

Comment:
If this is possibly null, are we sure that the JS side of things is going to provide a useful error to the consumer in that case?

Reply:
I updated screen to assert that `document.body` is not null so that this can be made non-nullable as well!